src/arch/riscv/linux/se_workload.cc

```diff
@@ -123,6 +125,368 @@ unameFunc32(SyscallDesc *desc, ThreadContext *tc, VPtr<Linux::utsname> name)
     return 0;
 }
 
+static inline void
+cpumask_set_cpu(unsigned int cpu,
+               uint64_t *dstp)
```

Comment:
style-nit: please fix the line break here and below. If the params fit within 79 characters, you don't need to break the line.

```cpp
cpumask_set_cpu(unsigned int cpu, uint64_t *dstp)
```